addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -103,9 +103,16 @@ const Wysiwyg = Widget.extend({
             function () {
                 self.odooEditor.observerUnactive();
                 const $field = $(this);
-                if (($field.data('oe-type') === "datetime" || $field.data('oe-type') === "date") && !$field.hasClass('o_editable_date_field_format_changed')) {
-                    $field.text($field.data('oe-original-with-format'));
-                    $field.addClass('o_editable_date_field_format_changed');
+                if (($field.data('oe-type') === "datetime" || $field.data('oe-type') === "date")) {
+                    let selector = '[data-oe-id="' + $field.data('oe-id') + '"]';
+                    selector += '[data-oe-field="' + $field.data('oe-field') + '"]';
+                    selector += '[data-oe-model="' + $field.data('oe-model') + '"]';
+                    const $linkedFieldNodes = self.$editable.find(selector).addBack(selector);
```

Comment:
Why do you need `addBack` ? Can your field really be `$editable` itself? Or am I missing something?

Reply:
I am not sure if I remember why this was needed, or if it was actually.
As this was done in 11.0 back then and widely tested before merged, either
1. It was needed for the previous editor impl, but not now
2. It was needed for a specific flow and it still needed but I don't remember the case (as it works well for events page without it, I just tried)
3. It was not needed and was a leftover of a previous implementation attempt back then

Let's say it is not 2. and remove it, we can reintroduce it later if we find a bug because of it missing.

Reply:
Seeing https://github.com/odoo/odoo/pull/45555#discussion_r380661470, I am pretty sure it is not 3.
I guess `$editable` was/could be the clicked element back then

Reply:
In that case let's keep it, it's not like it's expensive anyway.